aten/src/ATen/native/hammerblade/Offload.cpp

```diff
@@ -48,6 +171,222 @@ void offload_tensor_scalar_impl(std::vector<Tensor> tensors,
 
 }
 
+<<<<<<< HEAD
```

Comment:
Ooops. you have unresolved merge conflicts here

Reply:
yep I deleted that!